ipynb/primary-processing/01-get_sequences_and_metadata.md

```diff
@@ -25,6 +25,7 @@ Now let's actually fetch the study data. `fetch_study` will only pull down acces
 >>> for accession in study_accessions:
 ...     new_samples = agu.fetch_study(accession, chp_path)
 >>> print "New samples received: %d" % new_samples
+New samples received: 0
```

Comment:
can you please scrub output from the notebooks? it looks like the changes to all of the notebooks were not intentional to include